services/agent/requirements/_base.in

```diff
@@ -8,7 +8,7 @@
 # intra-repo required dependencies
 --requirement ../../../packages/models-library/requirements/_base.in
 --requirement ../../../packages/settings-library/requirements/_base.in
---requirement ../../../packages/service-library/requirements/_base.in
+--requirement ../../../packages/service-library/requirements/_fastapi.in
```

Comment:
thx!

I would have done this in a separate PR but I guess the agent is not critical